pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -75,6 +75,31 @@ func (r *MigToken) CanI(client k8sclient.Client, namespace, resource, verb strin
 	return sar.Status.Allowed, nil
 }
 
+// Check if the user has `use` verb on the associated MigCluster
+func (r *MigToken) HasUseVerb(client k8sclient.Client) (bool, error) {
+	sar := authapi.SelfSubjectAccessReview{
+		Spec: authapi.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &authapi.ResourceAttributes{
+				Resource:     "migclusters",
+				Namespace:    r.Spec.MigClusterRef.Namespace,
```

Comment:
in order to be explicit should we populate `Group:` field as well?

Reply:
What does this do?

Reply:
Oh right, to be explicit that we are under migration ApiGroup. I can add it.